src/types.ts

```diff
@@ -3796,3 +3797,14 @@ export type VelocityFilterConfig = {
   rules: VelocityFilterConfigRule[];
   async?: boolean;
 };
+
+export type MoveChannelUpwardsParams<SCG extends ExtendableGenerics = DefaultGenerics> = {
```

Comment:
Will this always always be upwards?

Reply:
I believe so, yes - we do not handle other cases with this util specifically; I'm up for renaming it to something more generic if you prefer though